impl/src/test/java/org/ehcache/GettingStarted.java

```diff
@@ -69,6 +73,27 @@ public void testWikiExample() {
   }
 
   @Test
+  public void testTieredStore() throws Exception {
+    CacheConfiguration<Long, String> tieredCacheConfiguration = newCacheConfigurationBuilder()
+        .persistenceMode(CacheConfiguration.PersistenceMode.SWAP)
+        .withResourcePools(newResourcePoolsBuilder().with("heap").with("disk").build())
+        .addServiceConfig(new OnHeapStoreServiceConfig().storeByValue(true))
```

Comment:
I would leave the storeByValue at the `CacheConfiguration` level. This then makes declaring the services not needed and thus the cache building more straightforward.